CMakeLists.txt

```diff
@@ -167,126 +168,155 @@ IF(BUILD_STATIC_LIB)
 ELSE()
   ADD_LIBRARY(binaryen SHARED ${binaryen_SOURCES})
 ENDIF()
-TARGET_LINK_LIBRARIES(binaryen passes wasm asmjs emscripten-optimizer ir cfg support)
-INSTALL(TARGETS binaryen DESTINATION ${CMAKE_INSTALL_LIBDIR})
 
-INSTALL(FILES src/binaryen-c.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+set(_dependencies "passes;wasm;asmjs;emscripten-optimizer;ir;cfg;support")
+foreach(_dep ${_dependencies})
```

Comment:
the convention so far in this file seems to be ALL_CAPS for cmake stuff.  Probably better to follow the local convention for changes like this.